### ISSUE TITLE
GraphConvolution中dropout的设置问题

### ISSUE THREAD

**Issue by porvinci** [storyandwine/LAGCN#15]
`layers.py`文件中第22行
```
x = tf.nn.dropout(x, 1-self.dropout)
```
tf.nn.dropout的第2个参数含义为每个元素的值设置为0的概率，所以代码是否应该写为
```
x = tf.nn.dropout(x, self.dropout)
```

**Comment by storyandwine** [comment]
In TensorFlow 1.12, the second param is keep_prob, https://www.tensorflow.org/api_docs/python/tf/compat/v1/nn/dropout, which does not seem as tfv2.
